packages/caliper-fabric/test/utils/GenerateConfiguration.js

```diff
@@ -64,5 +79,23 @@ class GenerateTestConfiguration {
     _deepCloneObject(object) {
```

Comment:
I think this could go into "CaliperUtils" now .. there's quite a few instances of the deep clone pattern in the code base

Reply:
Would it make sense to raise an issue to move all of these types of calls into a single point or would you prefer to create in this PR and have the others fixed elsewhere ?

Reply:
actually just found a couple of other good 1 liners
```
let cloned = Object.assign({}, source);
```
or
```
let cloned = { ... source };
```
Think I will get rid of the method and use these instead